nexus/db-fixed-data/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "nexus-db-fixed-data"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
+
```

Comment:
it could be nice to have a `description` here saying what this crate is for?

Reply:
Added